README.md

```diff
@@ -44,6 +44,7 @@ This repository contains the reference implementation of the parser. Other imple
 We also know about the following community-driven implementations:
 
   - [`Fluent.Net`](https://github.com/blushingpenguin/Fluent.Net) by [@blushingpenguin](https://github.com/blushingpenguin). See [#93](https://github.com/projectfluent/fluent/issues/93) for more info.
+  - [`Linguini`](https://github.com/Ygg01/Linguini) by [@Ygg01](https://github.com/Ygg01)
```

Comment:
I think it would make sense to mention here that this is a .NET port as well, since it's not obvious from the name.

Reply:
Agreed. @Mailaender, if you could update the mention with such a clarifier, I'd be happy to accept this PR.